server/index.js

```diff
@@ -28,46 +25,25 @@ const router = (server) => {
   });
 
   server.post('/api/firm', (req, res, next) => {
-    let newFirm = req.params;
-
-    if (!_.find(users, (user) => user.user_id === newFirm.user_id)) {
```

Comment:
removing lodash dependency, and code not used as our examples and tests are happy path for now

Reply:
Why not just change to `users.find` instead of removing all of this? We _should_ add tests for error handling and we don't want to have to rewrite this.

Reply:
ive been following this mantra... to not have code thats not used or commented out.
since the git history will allow us to add it back if and when we add error handling tests.